src/components/InventoryGroupDetail/EdgeUpdateDeviceModal.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import ErrorState from '@redhat-cloud-services/frontend-components/ErrorState';
+import { resolveRelPath } from '../../Utilities/path';
+import { getNotificationProp } from '../../Utilities/edge';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+
+const EdgeUpdateDeviceModal = (props) => {
+  const dispatch = useDispatch();
+  const notificationProp = getNotificationProp(dispatch);
+  return (
+    <AsyncComponent
+      appName="edge"
+      module="./UpdateDeviceModal"
+      ErrorComponent={<ErrorState />}
+      navigateProp={useNavigate}
+      locationProp={useLocation}
+      notificationProp={notificationProp}
+      pathPrefix={resolveRelPath('')}
```

Comment:
Can we add an empty fallback? I don't like the fact the the whole view is shifted down when you click on Update button to accommodate a spinner which is rendered as a fallback before the modal loads.  

```suggestion
      pathPrefix={resolveRelPath('')}
      fallback={null}
```

Reply:
done